pkg/ansible/watches/watches.go

```diff
@@ -141,9 +142,45 @@ func (w *Watch) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	w.Finalizer = tmp.Finalizer
 	w.AnsibleVerbosity = getAnsibleVerbosity(gvk, ansibleVerbosityDefault)
 	w.Blacklist = tmp.Blacklist
+	w.addRolePlaybookPaths()
+
 	return nil
 }
 
+// addRolePlaybookPaths will add the full path based on the current dir
+func (w *Watch) addRolePlaybookPaths() {
+	w.Playbook = getFullPath(w.Playbook)
+	w.Role = getFullRolePath(w.Role)
+	if w.Finalizer != nil && len(w.Finalizer.Role) > 0 {
+		w.Finalizer.Role = getFullRolePath(w.Finalizer.Role)
+	}
+	if w.Finalizer != nil && len(w.Finalizer.Playbook) > 0 {
+		w.Finalizer.Playbook = getFullPath(w.Finalizer.Playbook)
+	}
```

Comment:
Do we validate that these paths exist when the operator is starting up? If not, it would be good to add that so that a user doesn't get an unexpected surprise the first time their CR is reconciled.

Reply:
The paths are checked in the`verifyAnsiblePath()` method which is called after the watch initialization.  
These checks have been made just when the CR is applied.

Reply:
Issue raised : https://github.com/operator-framework/operator-sdk/issues/2498